source/shared/cpp/ObjectModel/ParseUtil.cpp

```diff
@@ -77,6 +77,16 @@ namespace AdaptiveSharedNamespace
         return propertyValue.asString();
     }
 
+    std::string ParseUtil::GetString(const Json::Value& json, AdaptiveCardSchemaKey key, std::string defaultValue, bool isRequired)
```

Comment:
needs const ref here. it's weird that we are passing in an argument just so we can return it.